public/pages/Dashboard/utils/tableUtils.js

```diff
@@ -164,65 +178,99 @@ export const alertColumns = (
           dataSourceQuery
         );
         const monitorDefinition = monitorResp.resp;
+        // 2. If the monitor is created via visual editor, translate ui_metadata dsl filter to ppl filter
```

Comment:
Is there a particular reason to use PPL here?

Reply:
PPL supports log pattern operator, the same behavior as discover's log pattern. We planned to enhance alert summary with this log pattern ppl query